backend/app/api/api_v1/routers/actions.py

```diff
@@ -65,18 +65,14 @@ async def action_create(
         geography_id=action.geography_id,
         action_type_id=action.action_type_id,
         action_source_id=action.action_source_id,
-        # Modification date is set to date of document submission
-        mod_date=datetime.now().date(),
         documents=action.documents,
     )
 
     try:
         db_action = create_action(db, action_create)
     except Exception as e:
         if isinstance(e, IntegrityError):
-            raise HTTPException(
-                409, detail=f"Database integrity error, underlying={e.orig}"
-            )
+            raise HTTPException(409, detail=f"Database integrity error, underlying={e}")
```

Comment:
Might be useful to know what [orig](https://docs.sqlalchemy.org/en/14/core/exceptions.html#sqlalchemy.exc.DBAPIError) is?

> The wrapped exception object is available in the [StatementError.orig](https://docs.sqlalchemy.org/en/14/core/exceptions.html#sqlalchemy.exc.StatementError.orig) attribute. Its type and properties are DB-API implementation specific.

Reply:
Actually, this doesn't matter for now. We need to revise the error-handling at some point 👍🏻

Reply:
I spent quite a while trying to get the type checker to accept that `.orig` but it stubbornly refused to believe it existed. The other option is to just disable the type check on that line...